### PR TITLE
feat: add e2e test for test create/update/delete

### PR DESCRIPTION
**Pull request by AnastasiaArcadia**
**Related to:**
https://oat-sa.atlassian.net/browse/AUT-122

**Description of changes**:
Sample E2E tests for item creation, edition, deletion.

**How to run locally:**
-  Checkout `feature/AUT-122/e2e` branch on [tao](https://github.com/oat-sa/tao-core/pull/2814)
- cd tao/views
- `npm install`
- rename `envs/env.sample` to `env.json` and update it with necessary config data
- `npm run cy:open` - to run in browser
- or `npm run cy:run` -  for headless execution

### CODE REVIEW

**Review by Silvia-loza — COMMENTED** [review]
The system works and runs tests smoothly. However some of the tests don't pass, not sure if correct behavior?

**Review by habemuscode — APPROVED** [review]
LGTM

**Review by bziondik — APPROVED** [review]
- [x]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by Silvia-loza — APPROVED** [review]
- [X] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Comment by mike-ionut-mihai-sandu-tao** [comment]
@jsconan and @martijn-tao we will merge the PR but please leave your comments because this is a ongoing and long initiative then for sure we will be able to take into account your comments next sprint 
so if no big objections ,  and the end of today we will merge and close this PR and the task attacked to it

File: views/cypress/tests/test.spec.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+     * Delete newly created tests after each step
+     */
+    afterEach(() => {
+       cy.deleteClassFromRoot(
+           selectors.root,
+           selectors.testClassForm,
+           selectors.deleteClass,
+           selectors.deleteConfirm,
+           newClassName
+       );
```
Comment (jsconan) [inline views/cypress/tests/test.spec.js:49]: **thought:** I wonder if this is needed, Unless it is part of the test scenario, but in this case it should take place inside it. Usually, e2e test should not perform a cleanup internally.

**Review by jsconan — COMMENTED** [review]
It looks ok globally, except for the internal cleanup, ~~but the test fails on my local instance~~

**Review by jsconan — APPROVED** [review]
It works well.

**Comment by jsconan** [comment]
**Review Checklist**
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**